shallow_backup/reinstall.py

```diff
@@ -64,6 +61,11 @@ def reinstall_packages_sb(packages_path):
 	"""
 	Reinstall all packages from the files in backup/installs.
 	"""
+	if not os.path.isdir(packages_path) or (os.path.isdir(packages_path) and 
```

Comment:
Taking advantage of short circuiting, we can shorten this to:

```suggestion
	if not os.path.isdir(packages_path) or not os.listdir(packages_path):
```

Reply:
`os.listdir()` throws a FileNotFoundError if the directory doesn't exist doesn't it?

Reply:
6947433a0765d4a3836a0f3062373ec56187c9fb